### ISSUE TITLE
Trigger Fields: The trigger button should get a focus stop

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4127]
**Describe the bug**
Making this as a seperate issue from https://github.com/infor-design/enterprise/issues/4085
It was mentioned in LM security scan that the trigger input should get a tab stop. And this should be the default.

We could save this for the refactor work and add to https://github.com/infor-design/enterprise/issues/579 but noting this as seperate.

**To Reproduce**
1. Go to https://master-enterprise.demo.design.infor.com/components/lookup/example-index.html
1. Tab to the first input
1. Tab again -> tab should be on the button
2. Also consider search field - the X should be tabbable too

**Expected behavior**
- [ ] Make the button have a tab stop
- [ ] Make a setting to disable this (non default)
- [ ] Look at other general accessibility around this

**Version**
- ids-enterprise: [v4.30 and below]

**Platform**
- All

**Additional context**
Other libs like material https://stackblitz.com/angular/nknyovevygv?file=src%2Fapp%2Fdate-range-picker-overview-example.html do make it

**Comment by ceszilla** [comment]
hi Tim. this is one of the Koch High Priority VPAT items. thanks.

**Comment by tmcconechy** [comment]
@ceszilla @pwpatton - i think we should have a quick call over this. It does mean that by default every form will have double the number of tabs to get through the page.

**Comment by ceszilla** [comment]
@tmcconechy @pwpatton for more info on the exact VPAT violation reported, please check https://jira.lawson.com/browse/LMCLIENT-34042 thank you

**Comment by tmcconechy** [comment]
Yeah really the questions are:

- can we wait on this to 5.0
- should we make the extra tab stop default
- should we add a setting

**Comment by ceszilla** [comment]
thanks Tim. ill try to find a common time my evening, your morning to quickly chat about this with Phillip.

**Comment by tmcconechy** [comment]
Im thinking we make an option similar to Mac where you can turn on extra tab stops to all items:

![Screen Shot 2021-02-01 at 1 20 07 PM](https://user-images.githubusercontent.com/814283/106503088-352c5500-6493-11eb-98fc-136445ad3633.png)

Then opt into this. In 5.0 we might want to make this default but allow you to turn it off.

**Comment by ceszilla** [comment]
had to move on a friday for common time, added @vonnyw . @pwpatton hope you're still available.

@tmcconechy just want to add that in triaging another violation this svg clear icon of the search text box was called out again, details below:

[Pattern: Clear button within search input is not accessible]
[Issue]
Navigating using a screen reader, "Clear" button within a search input field has multiple issues.
1. Tabindex="0" is implemented to make it focusable but at the same time it is not available to screen reader users as aria-hidden="true" is added. As a result, screen reader users cannot activate and clear the contents within the search input field.
2. It doesn't have an accessible name as a result it is announced as "Blank".
[User Impact]
Screen reader users are not aware of the clear button and may not interact with it appropriately.
[Code Reference]
<svg class="close icon" focusable="true" aria-hidden="true" role="presentation" tabindex="0"><use xlink:href="#icon-close"></use></svg>
[Recommendation]
Developers must ensure that they implement "clear" using <button> instead of an image. Add accessible name to the button such that it is announced with an appropriate name. Remove all the unnecessary event handlers and tabindex on <svg>.
[Compliant code example]
<button aria-label="Clear search">
<svg class="close icon" focusable="false" aria-hidden="true" role="presentation"><use xlink:href="#icon-close"></use></svg>
</button>

**Comment by tmcconechy** [comment]
@ceszilla i made https://github.com/infor-design/enterprise/issues/4815 for just the search part and gave that the priority

**Comment by ceszilla** [comment]
thanks @tmcconechy

**Comment by tmcconechy** [comment]
@ceszilla this part Isnt priority if the clear is fixed right?

**Comment by ceszilla** [comment]
@tmcconechy yup, 4815 is the one with the high priority. thanks

**Comment by tmcconechy** [comment]
This has been addressed on https://github.com/infor-design/enterprise/pull/5030 as far as i can tell

**Comment by jbrcna** [comment]
QA Passed
v4.50.0-beta2
https://4500-beta2-enterprise.demo.design.infor.com/components/lookup/example-index.html
![image](https://user-images.githubusercontent.com/56722184/113672432-5a834c80-96ea-11eb-8eca-1bc062e71f8e.png)

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
